addons/web/static/src/core/py_js/py_date.js

```diff
@@ -580,24 +639,70 @@ export class PyRelativeDelta {
      * @returns {PyDateTime}
      */
     static substract(date, delta) {
+        date = delta.applyAbsolute(date);
```

Comment:
maybe define a negate method and use it with add to define substract?
Also implement isEqual and isTrue?